### PR TITLE
New kbd Navpad 1.0

### PR DESCRIPTION
**Pull request by keyaki-namiki**
## Description

Added new keyboard Navpad 1.0

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* (None)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/yushakobo/navpad/10/rev0/config.h [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x3265
+#define PRODUCT_ID   0x0008 //not confirmed
```
Comment (MakotoKurauchi) [inline keyboards/yushakobo/navpad/10/rev0/config.h]:
```suggestion
#define PRODUCT_ID   0x0008
```

File: keyboards/yushakobo/navpad/10/rev1/config.h [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x3265
+#define PRODUCT_ID   0x0008 //not confirmed
```
Comment (MakotoKurauchi) [inline keyboards/yushakobo/navpad/10/rev1/config.h]:
```suggestion
#define PRODUCT_ID   0x0008
```

File: keyboards/yushakobo/navpad/10_helix_r/readme.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# navpad 1.0 with Helix (Right hand side)
+
+![10_helix_r](https://imgur.com/ehTI37h.jpg)
+
+Navpad connected with helix keyboard (Right Side)
+
+* Keyboard Maintainer: [yushakobo](https://github.com/yourusername)
```
Comment (MakotoKurauchi) [inline keyboards/yushakobo/navpad/10_helix_r/readme.md]:
```suggestion
* Keyboard Maintainer: [yushakobo](https://github.com/yushakobo)
```

File: keyboards/yushakobo/navpad/10/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# navpad 1.0
+
+![navpad_10](https://imgur.com/zxElDdW.jpg)
+
+Useful and easy to build numpad with nav-cluster
+
+* Keyboard Maintainer: [yushakobo](https://github.com/yushakobo)
+* Hardware Supported: NavPad 1.0 PCB, Pro Micro
+* Hardware Availability: https://shop.yushakobo.jp/navpad-10
```
Comment (MakotoKurauchi) [inline keyboards/yushakobo/navpad/10/readme.md]:
```suggestion
* Hardware Availability: https://shop.yushakobo.jp/products/navpad-10
```

File: keyboards/yushakobo/navpad/10_helix_r/readme.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+* Hardware Supported: Navpad 1.0 PCB and Helix PCB, Pro Micro
+* Hardware Availability: https://shop.yushakobo.jp/navpad-10
```
Comment (MakotoKurauchi) [inline keyboards/yushakobo/navpad/10_helix_r/readme.md]:
```suggestion
* Hardware Availability: https://shop.yushakobo.jp/products/navpad-10
```

**Review by MakotoKurauchi — CHANGES_REQUESTED** [review]

**Comment by keyaki-namiki** [comment]
Thank you for your reviews!

**Comment by MakotoKurauchi** [comment]
Thanks!

**Pull request merged**